spec/index.html

```diff
@@ -4557,8 +4549,9 @@ <h4>Descriptions of Resources</h4>
               </div>
             </div>
           </div>
-          <p>which includes the blank node closure for the <a data-cite="vcard-rdf#"
-                                                              class="inform">vcard</a> vocabulary vcard:N. Other possible mechanisms for deciding what
+          <p>which includes the blank node closure for the 
+            <a data-cite="vcard-rdf#" class="inform">vcard</a> vocabulary vcard:N.
```

Comment:
```suggestion
            <a data-cite="vcard-rdf#" class="inform">`vcard`</a> vocabulary `vcard:N`.
```

Reply:
Not the first change - it is a name. (And checking it's "vCard Ontology" but the latter is at risk of confusion with OWL and semantics in RDF specs - another "we are where we are").

Change made.